src/utils/version.js

```diff
@@ -4,4 +4,4 @@
 export const BREAKING_VERSION = 1;
 
 // Used for initialization message in chat, and userscript version
-export const VERSION = '1.1.7';
+export const VERSION = '1.1.8';
```

Comment:
What do you think about not updating the version, and I'll update it when I'm ready for release?
One less thing for contributors to worry about.

Reply:
Yeah keeping this to just you definitely makes sense lol